src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -1311,6 +1363,12 @@ private static void RunResultsCachingTestCase(ResultsCachingTestCase testCase, b
                 .BeEquivalentTo(runWithCaching.Invocations?[0].ToolExecutionNotifications);
         }
 
+        private static void EnhanceOptions(TestAnalyzeOptions current, TestAnalyzeOptions enhancement)
```

Comment:
>EnhanceOptions [](http://example.com/codeflow?start=28&length=14)

This method will be used to enhance the options with custom information. That will enable us to test multiple combinations with the same test, just changing the enhancement variable.